### ISSUE TITLE
option for "lmerTest" style anova tables with main and interactive effects

### ISSUE THREAD

**Issue by mluerig** [strengejacke/sjPlot#451]
maybe I missed it, but does sjPlot have an option to create anova tables that include the interactive effects, as shown below?

![image](https://user-images.githubusercontent.com/15648068/51970735-edbff800-2477-11e9-98b5-1559430205b9.png)

**Comment by strengejacke** [comment]
No, this is currently not possible. If `anova()` returns a data frame, you could at least use `tab_model()`.

**Comment by strengejacke** [comment]
Sorry, I meant `tab_df()`, not `tab_model()`.

**Comment by ABindoff** [comment]
This is how I do it,
tab_anova <- function(model, digits = 1, eps = .001){
  sjPlot::tab_df(broom::tidy(anova(model)) %>% 
                   mutate(p.value = format.pval(p.value, digits = digits, eps = eps)), digits = digits)
}
